etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/RSEndpointClient.java

```diff
@@ -0,0 +1,15 @@
+package gov.hhs.cdc.trustedintermediary.etor;
+
+import gov.hhs.cdc.trustedintermediary.external.reportstream.ReportStreamEndpointClientException;
+import javax.annotation.Nonnull;
+
+public interface RSEndpointClient {
+
+    String getRsToken() throws ReportStreamEndpointClientException;
+
+    String requestWatersEndpoint(@Nonnull String body, @Nonnull String bearerToken)
+            throws ReportStreamEndpointClientException;
+
+    String requestHistoryEndpoint(@Nonnull String submissionId, @Nonnull String bearerToken)
```

Comment:
Can we take out these @Nonnull annotations? They apparently only work 1 call down and only within one's IDE.  Feels a bit like giving folks a false sense of security which could lead to issues.

Reply:
That sounds good to me

Reply:
Done